### ISSUE TITLE
[Intermittent] Doorhanger either freezes or doesn't stops listening after 6 seconds if no command was spoken

### ISSUE THREAD

**Issue by alexandra-martin** [mozilla/firefox-voice#154]
### Steps to reproduce:
1. Click on the mic icon from the browser toolbar.
2. Wait 6 seconds.
3. Observe page.

### Expected result:
A notification is displayed indicating that no command was made.

### Actual result:
Doorhanger freezes after 6 seconds at the "One second..." stance.

### Notes:
Reproduced on Mac 10.14.6 with Firefox Nightly 70.0a1 (64-bit).
Another bug with the same steps is when the doorhanger can continue to listen and doesn't stops on its own (second gif).

![myFile](https://user-images.githubusercontent.com/54401091/64025568-a13fb680-cb45-11e9-8a24-1db106a20a66.gif)

![myFile (2)](https://user-images.githubusercontent.com/54401091/64027023-92a6ce80-cb48-11e9-94f6-40cd9455514a.gif)

**Comment by ianb** [comment]
I _think_ this was fixed in #284

**Comment by alexandra-martin** [comment]
Verified on Mac 10.14.6 and Win10x64 with Firefox Nightly 71.0a1 (64-bit).
On Mac I found that intermittently, the "Doorhanger" can continue to listen and doesn't stops on its own, but I can't seem to reproduce it regularly to know for sure out of 10 times how many times it is reproduced. (first gif)
Also found a **new actual result** as followed (second gif):

- The "Doorhanger" can close without displaying the processing and the "Sorry, there was an issue" states.

Reproduced intermittently 1/10, 2/10 or even 3/10 times on Mac 10.14.6.
